app/adaptive_tessellation/Parameters.h

```diff
@@ -105,6 +105,7 @@ struct Parameters
         const nlohmann::json& js,
         bool flush = false) // flush should force file output immediately, but will be slow for
                             // per-operation things
+        const
```

Comment:
wasn't your code but i wish this was in a cpp :(

Reply:
I also moved that to a cpp file. It might make sense to move other stuff to .cpp too but I'll keep it like that for now.